tests/test_ieee_backend/test.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 set -e
-verificarlo-c --save-temps -O0 test.c -o test
+verificarlo-c -ffp-contract=off --save-temps -O0 test.c -o test
```

Comment:
I think `-ffp-contract=off` should be added to the default verificarlo flags in the driver script.  Until we fix FMA support we need it everywhere and not only on this test.

Reply:
Comment for the record, I will validate your solution.
This is what I had done first but I changed my mind. 
First because the idea was to open a separate issue for it that will be solved after @yohanchatelain PR to support FMA (i will open it anyway to remind us to remove this flag during the PR)
Second because I am not sure of other negative aspect of forcing -ffp-contract=off
What makes me think it is fair to reconsider the situation as you suggest is that even if outside the test_ieee, the fact of not instrumenting the FMA will not cause a crash, not instrumenting some of the operation might be a worse behavior than changing a bit the arithmetic with -ffp-contract....
The only right fix is to support FMA... (and reduce)

Reply:
BTW I just tested your version successfully.